lib/package-updates-status-view.js

```diff
@@ -120,6 +121,22 @@ export default class PackageUpdatesStatusView {
   }
 
   updateTile () {
+    const disabledPackages = []
+    if (this.updates.length){
+      for (let index = 0; index < this.updates.length; index++) {
+        const update = this.updates[index]
+        if (atom.packages.isPackageDisabled(update.name)) {
+          disabledPackages.push(update)
+        }
+      }
+      for (let index = 0; index < disabledPackages.length; index++) {
+        if (this.updates.includes(disabledPackages[index])){
+          const location = this.updates.indexOf(disabledPackages[index])
+          this.updates.splice(location,1)
+        }
+      }
+    }
+
```

Comment:
I _believe_ that these two loops can be replaced with:

```
const updatesWithoutDisabled = this.updates.filter(update => !atom.packages.isPackageDisabled(update.name))
```

Reply:
Thank you, I refactored the code using filter instead of for-loops and no the specs seem to be working.